converters/fromZigbee.js

```diff
@@ -6435,6 +6435,28 @@ const converters = {
             return result;
         },
     },
+    develco_genbin: {
+        cluster: 'genBinaryInput',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const result = {};
+            if (msg.data.hasOwnProperty('outOfService')) {
+                result.out_of_service = (msg.data['outOfService']===1);
+            }
+            if (msg.data.hasOwnProperty('presentValue')) {
+                result.present_value = (msg.data['presentValue']===1);
```

Comment:
This name is a bit abstract, what does it mean?

Reply:
@Koenkk Removed `outOfService` & `presentValue`